rallyrolebot/cogs/purchase_cog.py

```diff
@@ -0,0 +1,97 @@
+import json
+import sys
+import traceback
+
+import discord
+from discord.ext import commands, tasks
+from discord.utils import get
+
+import data
+import rally_api
+import validation
+import errors
+
+from cogs import update_cog
+
+from constants import *
+from utils import pretty_print
+
+
+class PurchaseCommands(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def cog_after_invoke(self, ctx):
+        await pretty_print(
+            ctx, "Command completed successfully!", title="Success", color=SUCCESS_COLOR
+        )
+
+    @errors.standard_error_handler
+    async def cog_command_error(self, ctx, error):
+        # All other Errors not returned come here. And we can just print the default TraceBack.
+        print("Ignoring exception in command {}:".format(ctx.command), file=sys.stderr)
+        traceback.print_exception(
+            type(error), error, error.__traceback__, file=sys.stderr
+        )
+
+    @commands.command(
+        name="purchase",
+        help="Display purchase message",
+    )
+    async def purchase_message(self, ctx):
+        coin = data.get_default_coin(ctx.guild.id) or ""
+        default = f"You can purchase by using a Credit/Debit card or a number of different Crypto Currencies! Buying earns rewards, supports the community, and you can even get vip Status! (hint: there's a secret VIP room for those who hold X # of ;) https://www.rally.io/creator/{coin}"
+        message = data.get_purchase_message(ctx.guild.id) or default
+        await ctx.send(message)
+
+    @commands.command(
+        name="set_purchase_message",
+        help=" <message> Message displayed when user types the purchase command",
+    )
+    @validation.owner_or_permissions(administrator=True)
+    async def set_purchase_message(self, ctx, *args):
```

Comment:
Use the discord.py keyword argument notation here

```suggestion
    async def set_purchase_message(self, ctx, *, message):
        # message will be interpreted as a single string
```